include/zephyr/net/zperf.h

```diff
@@ -49,9 +49,9 @@ struct zperf_results {
 	uint32_t nb_packets_lost;
 	uint32_t nb_packets_outorder;
 	uint32_t total_len;
-	uint32_t time_in_us;
+	uint64_t time_in_us;
 	uint32_t jitter_in_us;
-	uint32_t client_time_in_us;
+	uint64_t client_time_in_us;
```

Comment:
Hmm, are you sure this works as expected, how well this is tested? I am asking because I recall that the results struct values are passed between zephyr and iperf program running in Linux and there could be some issues if we change the data type of the variables.

Reply:
Hello,
Sorry for the late answer, I changed it in zephyr version 2.7.2 and it work (tested). I tried to apply it to this version and haven't tested it but there seems to be some change in zperf (from commit cd4f7cbc6162cc51d3ba3ed1a23a78d49b690f3b) so it doesn't work. I will update and test for this PR.